pkg/controller/deployment_inplace_test.go

```diff
@@ -0,0 +1,1305 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package controller
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+
+	machinev1 "github.com/gardener/machine-controller-manager/pkg/apis/machine/v1alpha1"
+)
+
+var _ = Describe("deployment_inplace", func() {
+	Describe("syncMachineSets", func() {
+		type setup struct {
+			oldMachineSetReplicas                  int32
+			oldMSMachinesMoveToNewMS               int32
```

Comment:
```suggestion
			oldMSMachinesMovedToNewMS              int32
```
Please consider this rename